newsfragments/3975.minor

```diff
@@ -0,0 +1 @@
+Fixes truthy conditional in status.py
```

Comment:
This is fine, but just FYI the `.minor` newsfragments don't  render any text in the final changelog so we usually just leave them blank.

Reply:
Good to know I won't include them in the future.